### DISCUSSION TITLE
Miner Legacy Fork Fix не поддерживает более 64 виртуальных ядер. Майнинг на CPU будет отключен.

### DISCUSSION THREAD

**Discussion by mva19831** [angelbbs/NiceHashMinerLegacy#203] [General]
Можно ли это как-то исправить и чтобы майнинг на процессоре работал?
![MLFF_Nicehash](https://github.com/user-attachments/assets/b4677fe2-68ca-44ba-b26d-f336811a123d)

**Comment by angelbbs** [comment]
Fixed on next release
